Add instructions for nodejs debugging locally (#17846)
kyma-project/kyma: components/function-runtimes/nodejs/README.md

## How to locally run and debug the Node.js Function and runtime

* Copy `package.json` from the desired Node.js version
* Create the `function` directory with `handler.js` and `package.json`
* Install dependencies from the runtime and Function:

> ```bash

Reviewer: ```suggestion
   ```bash
```

> npm install

Reviewer: ```suggestion
   npm install
```

> npm install function/

Reviewer: ```suggestion
   npm install function/
```

> ```

Reviewer: Can you add three blank spaces in line 9?

* Run Function from the terminal.

> ```bash

Reviewer: ```suggestion
   ```bash
```

> npm start

Reviewer: ```suggestion
   npm start
```

> ```

Reviewer: Can you add three blank spaces in line 13?

> ## How to run/debug Nodejs function and runtime locally

Reviewer: ```suggestion
## How to locally run and debug the Node.js Function and runtime
```

> * Copy `package.json` from desired nodejs version * 

Reviewer: ```suggestion
* Copy `package.json` from the desired Node.js version
```

> * Create directory `function` with `handler.js` and `package.json`

Reviewer: ```suggestion
* Create the `function` directory with `handler.js` and `package.json`
```

> * Install dependencies from runtime and function:

Reviewer: ```suggestion
* Install dependencies from the runtime and Function:
```

```bash
npm install
npm install function/
```

> * Set proper environments variables TODO: check if it's needed

Reviewer: ```suggestion
* Set proper environment variables TODO: check if it's needed
```

```bash
export MOD_NAME=handler
export FUNC_HANDLER=main
```

> * Run function from terminal.

Reviewer: ```suggestion
* Run Function from the terminal.
```
